download_dependencies_linux.sh

```diff
@@ -53,14 +53,10 @@ fi
 
 copy_wget() {
 echo "wget: $2 <= $1"
-f1=$(basename $1)
-f2=$(basename $2)
-cd $(dirname $2)
+f=$(basename $2)
+d=$(dirname $2)
+cd $d
 wget -q -L -N $1
-# wget has no true equivalent of curl's -o option.
-# Different versions of wget handle (or not) % escaping differently.
-# A URL query is the only reason why $f1 and $f2 should differ.
```

Comment:
looks like you're using an old version of BuildUpdate. This extra logic is needed in both scripts.